### ISSUE TITLE
Validation errors are not displayed when empty form is submitted - Composition API

### ISSUE THREAD

**Issue by GMKR** [inkline/inkline#329]
* **What is the expected and current behavior?**
If an empty form with validation is submitted, error messages are not displayed, and the `submit` event is not triggered. 

When an empty form is submitted, all validations should run and proper error messages should be displayed

* **Steps to reproduce**

Component
```
<script setup lang="ts">
import { useForm } from "@inkline/inkline/composition-api";

const email = ref<string>();
const password = ref<string>();
const isLoading = ref<boolean>(false);

const validationSchema = {
  email: {
    validators: [{ name: "email" }],
  },
  password: {
    validators: [{ name: "required" }, { name: "minLength", value: 8 }],
  },
};
const form = useForm(validationSchema);

function handleSubmit() {
  isLoading.value = true;
  console.log(email.value, password.value);
  setTimeout(() => {
    isLoading.value = false;
  }, 2000);
}
</script>

<template>
  <i-form v-model="form" @submit="handleSubmit">
    <i-form-group required>
      <i-form-label>Email</i-form-label>
      <i-input v-model="email" name="email" placeholder="email address" />
      <i-form-error for="email" />
    </i-form-group>
    <i-form-group required>
      <i-form-label>Password</i-form-label>
      <i-input v-model="password" name="password" placeholder="password" />
      <i-form-error for="password" />
    </i-form-group>
    <i-form-group>
      <i-button type="submit" :loading="isLoading">
        Sign Up
        <template #loading>
          <i-loader color="dark" class="_margin-right:1/2" />
          Please wait..
        </template>
      </i-button>
    </i-form-group>
  </i-form>
</template>

```

* **Please tell us about your environment:**
    - Version: 3.1.8
    - Browser: Brave
    - Language: TypeScript
    - Environment - Nuxt 3

**Comment by 80982proton** [comment]
I have the same problem, is there a way around it? 

![image](https://user-images.githubusercontent.com/125216699/232089412-9f5a2bf2-d895-418c-bc3b-87f7ff67213b.png)

**Comment by alexgrozav** [comment]
Fixed in version 4.4.0. Now there's also a `validate` function being returned by `useForm` that can be used to validate the form on demand.

**Comment by alexgrozav** [comment]
@all-contributors add @GMKR for bug report.
